include/roboteam_utils/HalfLine.h

```diff
@@ -0,0 +1,36 @@
+#ifndef RTT_ROBOTEAM_UTILS_SRC_HALFLINE_H_
+#define RTT_ROBOTEAM_UTILS_SRC_HALFLINE_H_
+
+#include "Line.h"
+#include "Vector2.h"
+
+namespace rtt {
+/**
+ * The HalfLine class represents a line that is semi-infinite long, i.e. it has an end point where the line stops and an direction in which it is infinite long.
+ *
+ * @author Haico Dorenbos
+ * @since 2020-04-22
+ */
+class HalfLine {
+    private:
+        const Vector2 ending; // The location where the HalfLine ends
```

Comment:
I think this is confusing w.r.t the other line-like classes. I would rather see 'start' being used as for both other classes we defined direction as (end-start) so a vector going from start to end. Also when considering coneptually this as a ray you would typically start at t=0 and go to infinity, rather than going from infinity to a finite point.

Reply:
I was arguing on both sides what was the best name for it, since you prefer to name it start I will rename it to start.